### PR TITLE
[ZH] Translation of 09-3.md, 07-2.md, fix 07-1.md

### PR DESCRIPTION
**Pull request by JonathanSum**
Adding and Fixing:
[ZH]Week9 part3 Chinese Simplified Translations.

[Adding]Adding YouTube links for week6 part2 according to the English version(the one that I translated).

[Changing and fixing some translation for weeks6.md, week 7.md, week6 par2, week 7part1 ]

Practice: 动手做
Latent: 潛在

Changing实践 to 动手做 for the word "practice" to have a consistent translate as other people's translation
Changing some word's translation with this: latent to be 潛在 instead of 隱. That is because 潛在 means hidden but exists. 隱 means hidden only.

[EN Fixing] changing regularized to regulariser for Week 7 part1.

### CODE REVIEW

**Comment by Atcold** [comment]
OMG! These are 4 PRs in one! LOL.

File: docs/en/week09/09-3.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ $$
 As the discriminator becomes more confident, $D(\vect{x})$ gets closer to $1$ and $D(\vect{\hat{x}})$ gets closer to $0$. This confidence moves the outputs of the cost network into flatter regions where the gradients become more saturated. These flatter regions provide small, vanishing gradients that hinder the generator network's training. Thus, when training a GAN, you want to make sure that the cost gradually increases as you become more confident.
 
 
-### 3. Mode collapse
+### 3. Model collapse
```
Comment (Atcold) [inline docs/en/week09/09-3.md:75]: Nope. This is *mode collapse*. See [mode](https://www.oxfordlearnersdictionaries.com/definition/english/mode?q=mode).

**Comment by Atcold** [comment]
Next time, please, send each one separately.
One PR per `[xx]`, where `xx` is a given language.

File: docs/en/week09/09-3.md (hunk 1 above)
Comment (JonathanSum) [reply docs/en/week09/09-3.md]: I changed it.

**Comment by JonathanSum** [comment]
For this pull request, I actually translated Week7 part1 again, which I translated.
In addition, I just added Week7 part2. I guess GitHub does not allow me to open another pull request when this one is waiting to be merge.

**Comment by Atcold** [comment]
Thanks!

**Pull request merged**